actions/setup-token/.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
NOTE: shall we call all the token actions `token-***` instead of `***-token`?

Reply:
Is there an advantage to one or the other naming scheme?

Reply:
Offhand I don't see one, so I think ***-token sounds more right to me

If there's a benefit to a common prefix, I would guess it would be something like delegator-***